python/sglang/srt/layers/quantization/fp8.py

```diff
@@ -487,7 +488,7 @@ def create_weights(
 
         if self.quant_config.is_checkpoint_fp8_serialized:
             params_dtype = (
-                torch.int32
+                torch.uint32
```

Comment:
QQ why change this? It's also be used by NVIDIA GPU

Reply:
This is a case for uint32 packed int4 in serialized checkpoint case (within the flag), else case is generic case of OCP/NV data read from checkpoint.